aries-site/src/examples/components/checkbox/CheckBoxSimpleExample.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, FormField } from 'grommet';
+
+export const CheckBoxSimpleExample = () => {
+  const [checked, setChecked] = useState(false);
+  return (
+    <Box align="center" pad="large">
+      <FormField htmlFor="checkbox" label="label">
+        <CheckBox
```

Comment:
```suggestion
        <CheckBox
        id="checkbox"
```
Need an `id` with same value as the `htmlFor` to associate the labels.